test/repl.jl

```diff
@@ -32,29 +41,24 @@ ccall(:jl_exit_on_sigint, Void, (Cint,), 0)
 # this should make sure nothing crashes without depending on how exactly the control
 # characters are being used.
 if !Sys.iswindows() || Sys.windows_version() >= Sys.WINDOWS_VISTA_VER
-    stdin_write, stdout_read, stderr_read, repl = fake_repl()
-
+    fake_repl() do stdin_write, stdout_read, repl
```

Comment:
why not replace the vista check with this line so the indentation is still right without having to change the block